backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java

```diff
@@ -1,16 +1,30 @@
 package gov.cdc.usds.simplereport.config.authorization;
 
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
 public enum OrganizationRole {
-    USER("Users"), 
-    ADMIN("Admins"); // ?
+    ENTRY_ONLY("Test-entry users",
+            EnumSet.of(UserPermission.START_TEST, UserPermission.UPDATE_TEST, UserPermission.SUBMIT_TEST)),
```

Comment:
to do a search for patients and add them to the queue, would these users also need the READ_PATIENT_LIST permission?

Reply:
They need to be able to search users through the test-start dialog, yes. The fact that this functionally allows them full access to the patient list is ... one of those funny quirks of the requirements. 😕